.goreleaser.yaml

```diff
@@ -16,14 +16,14 @@ source:
   name_template: '{{ .ProjectName }}-{{ .Tag }}-source'
 archives:
   # Default template uses underscores instead of -
-  - name_template: "{{ .ProjectName }}-{{ .Tag }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
-    # TODO: this is deprecated -> https://goreleaser.com/deprecations#archivesreplacements
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - name_template: >-
```

Comment:
well, it may be deprecated but it looks nicer the old way to me lol

Reply:
indeed, it was much better before IMO